feedback/migrations/0021_contexttag.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('question_key', models.CharField(max_length=128, verbose_name='Question key')),
-                ('response_value', models.CharField(help_text='Indicate for which response values this tag should be displayed. This can be a regex pattern.', max_length=16, verbose_name='Response value')),
+                ('response_value', models.CharField(help_text='Indicate for which response values this tag should be displayed. This can be a regex pattern.', max_length=128, verbose_name='Response value')),
```

Comment:
Good to know, in that case, I'll change the original commit to include this. Although I was also thinking whether the commits "Add context tag" and "Improve context tag" should just be squashed. I left them separate for now in case the functionality without regex pattern matching is wanted (probably not), so the two commit could probably be squashed as well.

Reply:
I suppose Tuomas's comment was related to migrations that are new in this pull request. Then (like he said), it is nicer (but not strictly required) to clean up the git history so that the pull request adds only one version of the migration instead of changing it between the new commits.

However, there are some situations when it is necessary to modify migrations that have been added to the project a long time ago. Mainly, if some API has been changed so that the old migration simply crashes on new Django versions. (E.g., the import path has been changed and the import statements must be changed so that they work on the new Django version.)

Reply:
Yes, my comment was about editing migration such that the resulting database is different. You can do whatever changes you want to the migration code as long as the resulting database doesn't change (as long as other good practices are followed).